acme/acme/client.py

```diff
@@ -19,8 +19,8 @@
 
 logger = logging.getLogger(__name__)
 
-# https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning
-if six.PY2:
+# Python does not validate certificates by default before version 2.7.9
+if sys.version_info < (2, 7, 9):
```

Comment:
Travis fails because this line is not covered. Could you please add `  # pragma: no check` at the end of this line (note two prefix spaces)